core/foundation/inc/ROOT/RIntegerSequence.hxx

```diff
@@ -15,6 +15,7 @@
 #define ROOT_RIntegerSequence
 
 #include <RConfigure.h>
+#include "ROOT/span.hxx"
```

Comment:
This should be `"ROOT/RSpan.hxx"`. And yes, we need to hide `span.hxx`!

Reply:
thnx! you are right!